app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/FormApproveOrReject.tsx

```diff
@@ -38,7 +38,7 @@ const FormApproveOrReject = ({
           </p>
         </Alert>
       )}
-      <Alert variant="secondary">
+      <Alert variant="secondary" data-testid="review-request-alert">
```

Comment:
I think it's worth adding a test if the rollout is false, `review-request-alert` is not rendered in that case

Reply:
The review request alert is the box that says "email@email.com requested to {...}", so it _should_ be rendered in all cases :D I'll add this to the other tests!

Reply:
https://github.com/mozilla/experimenter/pull/8317/files#r1108878470